calicoctl/calico_ctl/node.py

```diff
@@ -262,8 +262,10 @@ def node_start(node_image, runtime, log_dir, ip, ip6, as_num, detach,
     if not os.path.exists(log_dir):
         os.makedirs(log_dir)
 
-    # The format of the authority string has already been validated.
+    # The format of the authority and endpoints strings have already been
+    # validated.
     etcd_authority = os.getenv(ETCD_AUTHORITY_ENV, ETCD_AUTHORITY_DEFAULT)
+    etcd_endpoints = os.getenv(ETCD_ENDPOINTS_ENV, None)
```

Comment:
I think this needs to be `""`.  When added to the `etcd_envs` below, it will add it as `"None"`, rather than `""`, then Felix may interpret `"None"` as a path and fail.